.github/workflows/test_suite.yml

```diff
@@ -37,36 +37,67 @@ jobs:
     runs-on: ubuntu-latest
     needs: [ codechecks ]
     strategy:
+      max-parallel: 1
       matrix:
         browser: [ chrome, firefox ]
         python-version: [ pypy3, 3.6, 3.7, 3.8 ]
 
+    services:
+      webdriver:
+        image: quay.io/redhatqe/selenium-standalone:latest
+        ports:
+          - 4444:4444
+          - 8080:8080
+        options: "--shm-size=2g"
```

Comment:
I believe tests failed because docker ran the container without `--network host`